tools/darwin/Support/copyframeworks-darwin_embedded.command

```diff
@@ -95,27 +95,6 @@ echo "Checking addons *.so for dylib dependencies"
 check_xbmc_dylib_depends "$XBMC_HOME"/addons "*.so"
 
 echo "Checking xbmc/DllPaths_generated.h for dylib dependencies"
-for a in $(grep .dylib "$BUILD_ROOT"/xbmc/DllPaths_generated.h | awk '{print $3}' | sed s/\"//g) ; do
+for a in $(grep .so "$BUILD_ROOT"/xbmc/DllPaths_generated.h | awk '{print $3}' | sed s/\"//g) ; do
```

Comment:
Why?

Reply:
as discussed on slack: looks like it's actually a fix due to generated header having so since forever